ml-agents/mlagents/trainers/env_manager.py

```diff
@@ -9,6 +9,13 @@
 AllGroupSpec = Dict[AgentGroup, AgentGroupSpec]
 
 
+def get_global_agent_id(worker_id: int, agent_id: int) -> str:
```

Comment:
I think this might be better in a utilities class ? But maybe not, I think it is common in Python to have methods lying around.

Reply:
I did debate for a while as to where this would live. Currently we don't have a utilities class, we'd have to create one just for this. What about putting it into the brain_conversion_utils file, or the action_info file?